ores/wsgi/util.py

```diff
@@ -35,23 +37,147 @@ def read_bar_split_param(request, param, default=None, type=str):
                          .format(param, str(e)))
 
 
+def format_error(error):
+    error_type = error.__class__.__name__
+    message = str(error)
+
+    return {'error': {'type': error_type, 'message': message}}
+
+
+def build_score_request(scoring_system, request, context_name=None, rev_id=None,
```

Comment:
I like having it here because there's a few different ways we build requests.  E.g. from the command-line for running ORES offline.   

+1 for docstring